config/environments/production.rb

```diff
@@ -152,6 +152,11 @@
   PDF_STORAGE = "#{::Rails.root.to_s}/data/prod/pdfs"
 
   ###################################################################
+  # Location of the public and private key for the git user on the system
+  GITOLITE_SETTINGS = { public_key: "/home/git/git.pub",
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.